.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
- - 5.5
```

Comment:
- We do not use any feature that is different in 5.5 and 5.6
- Our tests do not cover enough functions to be able at all to find any possible PHP bugs which might be in 5.5 but not in 5.6
- We need PHP for the builds only for composer. I would do that with the fastest one, that is PHP 7.
- We already test on PHP 7.